R/approach_regression_separate.R

```diff
@@ -269,7 +271,7 @@ regression.get_y_hat <- function(internal, model, predict_model) {
 regression.get_tune <- function(regression.model, regression.tune_values, x_train) {
   # Check that the regression model is a tidymodels object
   if (is.null(regression.model) || !"model_spec" %in% class(regression.model)) {
-    stop("`regression.model` must be a tidymodels object with class 'model_spec'. See documentation.")
+    cli::cli_abort("`regression.model` must be a tidymodels object with class 'model_spec'. See documentation.")
```

Comment:
```suggestion
    cli::cli_abort("`regression.model` must be a tidymodels object with class `model_spec`. See documentation.")
```

This is to match the format used in the other cli strings in this file.

Reply:
Fixed for all of cli_inform, not the others.